### ISSUE TITLE
[vapoursynth] Value Error: could not convert string to float

### ISSUE THREAD

**Issue by przkreft** [the-database/mpv-upscale-2x_animejanai#38]
I was able to upscale for some time and pretty quickly it stopped working after playing with profiles.
I have rife disabled in profiles.

Issue seems to be related to rife fixes.

![vapoursynth issue](https://github.com/user-attachments/assets/504cdd16-4cea-4ec8-b6de-657782c6be10)

**Comment by the-database** [comment]
This looks related to this issue https://github.com/the-database/VideoJaNai/issues/12, looks like there is a bug that happens in Windows localizations besides English.

**Comment by the-database** [comment]
Could you please try this build and let me know if the issue is still present? https://github.com/the-database/mpv-upscale-2x_animejanai/releases/tag/3.1.1

After installing the new build, launch the conf editor, make any changes, and then make sure that upscaling works with the changes you made.

**Comment by przkreft** [comment]
Yeah, sure :) my pleasure.

![image](https://github.com/user-attachments/assets/5bf6039b-581a-4424-b785-4acb15d55506)
It seems like still fails in same place. I marked as a proof new path and error (just to double confirm that I'm using new build).

**Comment by the-database** [comment]
Thanks for testing. I think I found the issue with the previous build. I replaced [release 3.1.1](https://github.com/the-database/mpv-upscale-2x_animejanai/releases/tag/3.1.1) with a new version, could you please try it and let me know if it's fixed?

**Comment by przkreft** [comment]
Unfortunately issue still persists.

E:\mpv-upscale-2x_animejanai-full-package-3.1.1\mpv-upscale-2x_animejanai-v3.1.1\animejanai\core

I've done hardcoding starting from line 36 which fixes issue for me:

`    min_width, min_height = [float(px.replace(',', '.')) for px in min_resolution.split("x")]
    max_width, max_height = [float(px.replace(',', '.')) for px in max_resolution.split("x")]

    if max_height == 0:
        max_height = float("inf")
    if max_width == 0:
        max_width = float("inf")

    max_fps = float(flat_conf[section].get(f"chain_{chain}_max_fps", "inf").replace(',', '.'))

    if max_fps == 0:
        max_fps = float("inf")

    return {
        "min_px": min_width * min_height,
        "max_px": max_width * max_height,
        "min_resolution": min_resolution,
        "max_resolution": max_resolution,
        "min_fps": float(flat_conf[section].get(f"chain_{chain}_min_fps", 0).replace(',', '.')),
        "max_fps": max_fps,
        "models": [
            read_config_by_chain_model(flat_conf, section, chain, i)
            for i in range(1, num_models + 1)
        ],
        "rife": parse_bool(flat_conf[section].get(f"chain_{chain}_rife", "no")),
        "rife_factor_numerator": int(
            float(flat_conf[section].get(f"chain_{chain}_rife_factor_numerator", 1).replace(',', '.'))
        ),
        "rife_factor_denominator": int(
            float(flat_conf[section].get(f"chain_{chain}_rife_factor_denominator", 1).replace(',', '.'))
        ),
        "rife_model": int(
            float(flat_conf[section].get(f"chain_{chain}_rife_model", 414).replace(',', '.'))
        ),
        "rife_ensemble": parse_bool(
            flat_conf[section].get(f"chain_{chain}_rife_ensemble", "no")
        ),
        "rife_scene_detect_threshold": float(
            flat_conf[section].get(f"chain_{chain}_rife_scene_detect_threshold", 0.150).replace(',', '.')
        ),
    }`

**Comment by the-database** [comment]
I found which Windows setting controls this (Time & language > Language & region > Regional format). I set my regional format to Polish to test this. I found that I could reproduce the issue on version 3.1.0, but I'm unable to reproduce the issue on 3.1.1, it seems to work correctly for me. Which regional format is your machine set to? Is it possible the version of 3.1.1 you tested was still the old version?

**Comment by przkreft** [comment]
Previously I unpacked new version 3.1.1 to same folder of let's say "previous" 3.1.1 and just overwritten files + copy pasted my config and fp32 onnx models. 

When I removed whole folder and unpacked new version it works fine.
Double checking -> downloaded and extracted 3.1.0, overwritten with files from new 3.1.1 (I was storing zip on hard drive) and also it works fine.

I'm not sure if the issue was with overwriting files from new version of 3.1.1 on top of let's say "previous" replaced 3.1.1 version or something different happened during my testing.

To sum up, now it works fine for me, Thank You very much!

**Comment by the-database** [comment]
Great to just hear, thanks for testing and confirming the fix!
